shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/entity/Entity.kt

```diff
@@ -38,4 +38,16 @@ data class MealWithDistance(
     val meal: Meal,
     @SerialName("distance")
     val distance: Double
+)
+
+@Serializable
+data class FirebaseAuthenticationResponse(
```

Comment:
Do the properties on this data class need to be nullable? Just looking at the ones above that aren't. Other than that, this PR looks nice to me! Nice and tidy